lib/src/over_react_test/jacket.dart

```diff
@@ -112,12 +120,21 @@ class TestJacket<T extends react.Component> {
     //     a DOM component (Element) - does not throw. The cast to `ReactComponent` - while not "sound", is harmless
     //     since it is an anonymous JS interop class - not a Dart type.
     if (!_isCompositeComponent && /*[1]*/!_isDomComponent) {
-      throw StateError(
-          'getInstance() is only supported when the rendered object is a composite (class based) component. '
-          'If you are rendering a function component, and are trying to:\n\n'
-          '1. Access an instance rendered by the function component: use `forwardRef` to pass refs through the tree.\n'
-          '2. Access / query for a DOM node rendered by the function component: try using\n'
-          '    queryByTestId(jacket.mountNode, yourTestId)');
+      throw StateError(over_react.unindent('''
+          getInstance() is only supported when the rendered object is a composite (class based) component.
+          
+          If you are rendering a function component, and are trying to:
+          
+          1. Access an instance of some other component rendered by it, then try either:
+              i. Wrapping the component in a class-based wrapper component (such as the `Wrapper` component exported by over_react_test):
+                  final jacket = mount(Wrapper()(
+                    YourFunctionComponent(),
+                  ));
+              ii. Using `forwardRef` or a ref prop to pass a ref through to the component you need
```

Comment:
is this `forwardRef` from over_react_test, or do you mean `uiForwardRef`?

Reply:
Ooh good catch; will update this